Description: Change page navigation buttons to match Callisto theme
 Quick and simple patch to use Callisto-style arrow buttons instead of
 the default NeXT-esque style. Also applies a blank area matching the
 FvwmButtons background to hide buttons unless there are multiple pages.
Origin: tsundoku custom patchset
Author: tsundoku <me@tsundoku.ne.jp>
Last-Update: 2019-02-25
--- a/wmsystemtray.xpm
+++ b/wmsystemtray.xpm
@@ -1,42 +1,89 @@
 /* XPM */
 static char * wmsystemtray_xpm[] = {
-"30 30 9 1",
-" 	c #020202",
-".	c #F6F2FE",
-"+	c #C6C6C6",
-"@	c #AEAAAE",
-"#	c #575557",
-"$	c #AAAAAA",
-"%	c #555555",
-"&	c #82828A",
-"*	c #000000",
-"                              ",
-" ............ + ............ +",
-" .@@@#@@@@@@@ + .@@@@@@@#@@@ +",
-" .$$%%$$$$$$$ + .$$$$$$$%%$$ +",
-" .@#%%%%#%%$$ + .@@%%%%#%%%$ +",
-" .$%%%%%%%%$$ + .$$%%%%%%%%$ +",
-" .@@%%$$@$$$$ + .@@$$$$@%%$$ +",
-" .$$$%$$$$$$$ + .$$$$$$$%$$$ +",
-" &            + &            +",
+"30 30 56 1",
+" 	c None",
+".	c #CECECE",
+"+	c #CDCDCD",
+"@	c #CCCCCC",
+"#	c #CBCBCB",
+"$	c #CACACA",
+"%	c #CFCFCF",
+"&	c #C9C9C9",
+"*	c #D7D7D7",
+"=	c #7C7C7C",
+"-	c #E3E3E3",
+";	c #E6E6E6",
+">	c #000000",
+",	c #E2E2E2",
+"'	c #E1E1E1",
+")	c #E1E1E2",
+"!	c #E1E2E2",
+"~	c #E1E2E1",
+"{	c #E2E2E1",
+"]	c #E2E1E2",
+"^	c #DDDDDD",
+"/	c #DDDCDD",
+"(	c #DCDCDD",
+"_	c #D8D8D8",
+":	c #D3D3D4",
+"<	c #D3D3D3",
+"[	c #D4D3D4",
+"}	c #D4D3D3",
+"|	c #CECECF",
+"1	c #CECFCE",
+"2	c #CFCFCE",
+"3	c #CFCECF",
+"4	c #CFCECE",
+"5	c #292929",
+"6	c #AEAEAE",
+"7	c #ACACAB",
+"8	c #ACACAC",
+"9	c #ACABAC",
+"0	c #ACABAB",
+"a	c #AAA9A9",
+"b	c #A9A9AA",
+"c	c #AAAAAA",
+"d	c #AAAAA9",
+"e	c #A7A7A8",
+"f	c #A8A7A8",
+"g	c #A8A8A7",
+"h	c #A8A7A7",
+"i	c #A7A7A7",
+"j	c #A7A8A8",
+"k	c #A5A6A5",
+"l	c #A5A5A5",
+"m	c #A6A5A5",
+"n	c #A6A6A5",
+"o	c #A6A6A6",
+"p	c #A6A5A6",
+"q	c #A3A3A3",
+"..............................",
 "++++++++++++++++++++++++++++++",
-"                              ",
-" ............ + ............ +",
-" .@@@*@@@@@@@ + .@@@@@@@*@@@ +",
-" .$$**$$$$$$$ + .$$$$$$$**$$ +",
-" .@********$$ + .@@********$ +",
-" .$********$$ + .$$********$ +",
-" .@@**$$@$$$$ + .@@$$$$@**$$ +",
-" .$$$*$$$$$$$ + .$$$$$$$*$$$ +",
-" &            + &            +",
 "++++++++++++++++++++++++++++++",
-"                              ",
-"             &+             &+",
-"  $$$*$$$$$$$.+  $$$$$$$*$$$.+",
-"  @@**@@@@@@@.+  @@@@@@@**@@.+",
-"  $********@$.+  $$********$.+",
-"  @********@@.+  @@********@.+",
-"  $$**@$$@@@$.+  $$@@@$$**@$.+",
-"  @@@*@@@@@@@.+  @@@@@@@*@@@.+",
-" &............+ &............+",
-"++++++++++++++++++++++++++++++"};
+"++++++++++++++++++++++++++++++",
+"@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@",
+"@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@",
+"##############################",
+"##############################",
+"$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$",
+"$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$",
+"%&&&&&&&&&&&&&*%&&&&&&&&&&&&&*",
+"&=============-&=============-",
+"&=;;;;;;>;;;;=-&=;;;;>;;;;;;=-",
+"&=,')!)>>)~{]=-&=,')!>>,)~{]=-",
+"&=^^^/>>>//(^=-&=^^^/>>>//(^=-",
+"&=____>>>____=-&=____>>>____=-",
+"&=:<[::>>}<}<=-&=:<[:>><}<}<=-",
+"&=|1%231>44%|=-&=|1%2>1144%|=-",
+"&=============-&=============-",
+"*-------------^*-------------^",
+"%&&&&&&&&&&&&&*%&&&&&&&&&&&&&*",
+"&5555555555555-&5555555555555-",
+"&5666666>66665-&56666>6666665-",
+"&578790>>88985-&57879>>888985-",
+"&5abcc>>>dccc5-&5abcc>>>dccc5-",
+"&5efgh>>>eijh5-&5efgh>>>eijh5-",
+"&5klmno>>plkl5-&5klmn>>nplkl5-",
+"&5qqqqqq>qqqq5-&5qqqq>qqqqqq5-",
+"&5555555555555-&5555555555555-",
+"*-------------^*-------------^"};
